inst/htmlwidgets/sankeyNetwork.js

```diff
@@ -118,7 +119,7 @@ HTMLWidgets.widget({
             .attr("text-anchor", "end")
             .attr("transform", null)
             .text(function(d) { return d.name; })
-            .style("font", options.fontsize + "px serif")
+            .style("font", options.fontsize + "px")
```

Comment:
`.style("font", options.fontsize + "px")`

should be

`.style("font-size", options.fontSize + "px")`